data/json/itemgroups/Locations_MapExtras/locations_commercial.json

```
@@ -894,7 +894,45 @@
     "type": "item_group",
     "id": "paintball_items",
     "//": "For paintball_field.json",
-    "items": [ [ "paintballgun", 55 ], [ "paintball", 45 ], [ "goggles_swim", 15 ] ]
+    "items": [
+      {
+        "distribution": [
+          { "item": "paintball", "prob": 125 },
+          { "item": "paintballgun", "prob": 100, "charges": [ 0, 50 ] },
+          { "group": "paintball_cloths", "prob": 30 }
+        ]
+      }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "paintball_cloths",
```

```suggestion
    "id": "paintball_clothes",
```
"Cloths" seems like an out-of-place name, at least to me.